templates/commands/render_test.go

```diff
@@ -478,47 +504,42 @@ func TestSafeRelPath(t *testing.T) {
 		},
 		{
 			name: "path_with_directories_succeeds",
-			in:   "a/b.txt",
-			want: "a/b.txt",
+			in:   "a" + slash + "b.txt",
+			want: "a" + slash + "b.txt",
 		},
 		{
 			name: "trailing_slash_succeeds",
-			in:   "a/b/",
-			want: "a/b/",
+			in:   "a" + slash + "b" + slash,
+			want: "a" + slash + "b" + slash,
 		},
 		{
 			name: "leading_slash_stripped",
-			in:   "/a",
+			in:   slash + "a",
 			want: "a",
 		},
 		{
 			name: "leading_slash_with_more_dirs",
-			in:   "/a/b/c",
-			want: "a/b/c",
-		},
-		{
-			name: "leading_slash_with_more_dirs",
```

Comment:
This was a duplicate test with the one above

Reply:
Ack